plugins/logger/tls_logger.cpp

```diff
@@ -96,6 +101,11 @@ void TLSLoggerPlugin::init(const std::string& name,
 
 Status TLSLogForwarder::send(std::vector<std::string>& log_data,
                              const std::string& log_type) {
+  // Skip sending status logs to remote server if disabled
+  if (log_type == "status" && FLAGS_tls_disable_status_log) {
```

Comment:
It's a tiny nit, but I assume this is somewhat faster as `FLAGS_tls_disable_status_log && log_type == "status"`

Reply:
Thank you for reviewing my PR, you are right, that will be faster and I just updated the code. Please have a check the update and let me know if you have any other comments. Thanks.